demo/packaged/README.md

```diff
@@ -0,0 +1,75 @@
+# Packaged SQL-only demos
+
+The SQL of packaged demo starts with a preamble to convey explicitly the
+title, name and description. This information is for example extracted by the
+Web Console for display on the demos overview.
+
+All the SQL files in `sql/` are packaged demos with every Feldera release.
+
+- **Preamble validation:**
+  ```bash
+  python3 validate-preamble.py sql/*.sql
+  ```
+
+- **Cargo run with demos:**
+  ```bash
+  cargo run --package=pipeline-manager \
```

Comment:
we should make this the default for someone running from the source repo

Reply:
Could you clarify with what you mean with "default"? As in, do you mean I change in the main repo README:
```
cargo run --bin=pipeline-manager --features pg-embed
```
... to:
```
cargo run --bin=pipeline-manager --features pg-embed -- --demos-dir demo/packaged/sql
```
?

Reply:
@snkas I think Gerd is suggesting that this be the default args if anyone runs `cargo run` in our repo.

Reply:
As the API server configuration is not just used by the `pipeline-manager` binary, and the binary should be standalone executable (not be dependent on a relative directory being present), adding it as a default seems not advantageous though? It could be added to the `scripts/start_manager.sh` script?

Reply:
I usually run the manager as 

```
cd feldera
cargo run --bin=pipeline-manager --features pg-embed
```

that's also what we suggest to users in the README. IMO in this case it should populate the demos without supplying any extra args.

Reply:
Making it default some value, let's say `Some(vec!["demo/packaged/sql"])`, will mean that it is not possible to have no demos directory anymore (due to the way argument parsing works). I would then drop the `Option<>` around it.

One alternative is to have an option "default_demos", and hardcode the demos in Rust.

Reply:
Let's go with it for now, I'll make the default `vec!["demo/packaged/sql"]` -- in all form factors we have demos in any case.

Reply:
I'll make that change as well, and just print out an error message in those cases.